wis2box-management/Dockerfile

```diff
@@ -70,6 +68,8 @@ RUN cd /app \
     # add wis2box user
     && useradd -ms /bin/bash wis2box
 
+RUN pip3 install https://github.com/david-i-berry/wis2box-migrations/archive/main.zip
```

Comment:
Are migrations not part of wis2box proper?

Reply:
no, this is an accidental commit, removing